docs/stdlib/json.rst

```diff
@@ -66,7 +67,7 @@ by casting a value into :eql:type:`json` or by using :eql:func:`to_json`:
     db> select <json>'hello world';
     {'"hello world"'}
 
-Anything in EdgeDB can be cast into :eql:type:`json`:
+Any value in EdgeDB can be casted to an :eql:type:`json` type as well:
```

Comment:
Yes to @colinhacks's suggestion. Also, I believe "cast" should be the past tense of the verb "cast."

```suggestion
Any value in EdgeDB can be cast to a :eql:type:`json` type as well:
```

Reply:
Just noticed the mention of "casted" below. 😅